src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,194 @@
+    if (this.email === undefined) {
+      this.context.stderr.write(renderError('No email specified. [-e,--email]'))
+      errorFound = true
+    }
+
+    if (errorFound) {
+      return 1
+    }
+
+    // Get AWS credentials
+    this.context.stdout.write('\n🔑 Getting AWS credentials...\n')
+    const credentials = await getAWSCredentials()
+    if (credentials === undefined) {
+      this.context.stdout.write(renderSoftWarning("No AWS credentials found, let's set them up!"))
+      await requestAWSCredentials()
+    } else {
+      this.credentials = credentials
+    }
+
+    // Get Lambda function configuration
+    this.context.stdout.write('🔍 Getting Lambda function configuration...\n')
+    const lambdaClientConfig: LambdaClientConfig = {
+      region: this.region,
+      credentials: this.credentials,
+    }
+    const lambdaClient = new LambdaClient(lambdaClientConfig)
+    if (this.functionName === undefined) {
+      throw new Error('Function name is undefined')
+    }
+    const config = await getLambdaFunctionConfig(lambdaClient, this.functionName)
+    const configStrColored = util.inspect(config, false, undefined, true)
+    const configStrUncolored = JSON.stringify(config, undefined, 2)
+
+    // Print config
+    this.context.stdout.write(`\n${configStrColored}\n`)
+    if (this.isDryRun) {
+      this.context.stdout.write('\n🚫 Configuration not sent because the command was ran as a dry run.\n')
+
+      return 0
+    }
+
+    // Write config to file
+    const folderPath = path.join(process.cwd(), '.lambda-flare-output')
+    if (!fs.existsSync(folderPath)) {
+      fs.mkdirSync(folderPath)
+    }
+    const filePath = path.join(folderPath, 'function_config.json')
+    fs.writeFileSync(filePath, configStrUncolored)
+
+    // Zip folder
+    const zipPath = path.join(folderPath, 'lambda-flare-output.zip')
+    const zipFolder = async (command: Command) => {
+      try {
+        const data = await fs.promises.readFile(filePath, 'utf8')
+        const zip = new JSZip()
+        zip.file('function_config.json', data)
+        const content = await zip.generateAsync({type: 'nodebuffer'})
+        await fs.promises.writeFile(zipPath, content)
+      } catch (error) {
+        command.context.stderr.write(renderError('Error reading file or writing zip file. Please try again.'))
+
+        return true
+      }
+
+      return false
+    }
+    if (await zipFolder(this)) {
+      return 1
+    }
+
+    // Send to Datadog
+    this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
+    const form = new FormData()
+    form.append('case_id', this.caseId)
+    form.append('flare_file', fs.createReadStream(zipPath))
+    form.append('operator_version', 7)
+    form.append('email', this.email)
+    const requestConfig = {
+      headers: {
+        ...form.getHeaders(),
+        'DD-API-KEY': this.apiKey,
+      },
+    }
+    await axios
+      .post('https://datad0g.com/api/ui/support/serverless/flare', form, requestConfig)
```

Comment:
Make this a constant, also, do we need `https://`?

Reply:
Yes looks like `https://` is required, post request fails if `https://` is not included